app/controllers/time_tracker_controller.rb

```diff
@@ -37,7 +37,11 @@ def create
   end
 
   def update
-    return render_js(:update, :unprocessable_entity) unless @current_timer_session.update(timer_params)
+    updated_params = timer_params
+
+    updated_params[:timer_end] = user_time_zone.now.asctime if updated_params[:timer_end].blank?
```

Comment:
Should be moved into the model perhaps?

Reply:
This is not nicely possible, because a timer session without end time is valid, right?